libs/components/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./AvatarFallback";
```

Comment:
Can this live at app level until its needed elsewhere?

Reply:
It could, but this'll be used wherever we use avatars which won't be just in team nav. This is also something I'd like to port to palette, I just need some input from design first.

Reply:
To keep it more general is a premature optimization I think, we should move it when it's actually shared.

Reply:
👍 okay, I moved it.